bundle/com.github.lruzicka.Needly.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.lruzicka.Needly.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <name>Needly</name>
+  <summary>openQA needle editor</summary>
+  <description>
+    <p>
+Needly is an openQA needle editor written in Python. It creates or modifies needles for the openQA tests. 
+    </p>
+    <p>
+The advantage of the editor is that it does not need openQA to be installed on the system. The needle files are all you need to work with them.
+    </p>
+  </description>
+  <url type="homepage">https://github.com/lruzicka/needly/tree/master/needly</url>
+  <url type="bugtracker">https://github.com/lruzicka/needly/issues</url>
+  <screenshots>
+    <screenshot type="default" priority="4">
+      <caption>Needly with a created openQA needle.</caption>
+      <image>https://github.com/lruzicka/needly/blob/master/needly/screens/needly.png</image>
+    </screenshot>
+  </screenshots>
+  <launchable type="desktop-id">com.github.lruzicka.Needly.desktop</launchable>
+  <provides>
+    <binary>needly</binary>
+  </provides>
+  <recommends>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <display_length side="longest" compare="ge">1400</display_length>
+  </recommends>
+  <releases>
+    <release version="2.5" date="2022-03-29">
```

Comment:
the tarball has version `2.5.5`

Reply:
I though I could cover the minor versions with a more general number. Will change it.